File: 2365-percentage-of-letter-in-string/percentage-of-letter-in-string.cpp
class Solution {
public:
    int percentageLetter(string s, char letter) {
        float ans = 0;
        for(char i : s){
            if(i == letter){
                ans++;
            }
        }
        ans = (ans * 100) / (float)s.size();
        return ans;
    }
};

File: 2365-percentage-of-letter-in-string/percentage-of-letter-in-string.java
class Solution {
    public int percentageLetter(String s, char letter) {
        int ans = 0;
        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) == letter){
                ans++;
            }
        }
        ans = (ans * 100) /  s.length();
        return ans;
    }
}